markovflow/sde/sde_utils.py

```diff
@@ -226,3 +242,219 @@ def func(x, t=None, A=A, b=b):
 
     drift_difference = 0.5 * tf.reduce_sum(drift_difference) * dt
     return drift_difference
+
+
+def gaussian_log_predictive_density(mean: tf.Tensor, chol_covariance: tf.Tensor, x: tf.Tensor) -> tf.Tensor:
+    """
+        Compute the log probability density for a Gaussian.
+    """
+    mvn = MultivariateGaussian(chol_covariance=chol_covariance)
```

Comment:
it is an odd object that first gets a cov and then a mean, but I guess there is a reason behind it

Reply:
It's an object already in Markovflow, which I just wanted to use to get predictive density.